airflow/www/views.py

```diff
@@ -263,19 +263,20 @@ def dag_to_grid(dag, dag_runs, session):
             TaskInstance.task_id,
             TaskInstance.run_id,
             TaskInstance.state,
-            TaskInstance.notes,
-            sqla.func.count(sqla.func.coalesce(TaskInstance.state, sqla.literal("no_status"))).label(
+            func.min(TaskInstanceNote.content).label("notes"),
```

Comment:
Using `min` on a string field seems.... unexpectec.

Reply:
so...
yes it's always weird when you see this but there's a reason
some dbs have `any_value` or `first`. that would make more sense here. but sqla no have this. `min` is proxy for "just give me something"
the alternative is what was there before, which is adding to the group by.  but that doesn't make much sense either (i asked brent, he said we don't need to be creating new groups based on the presence of notes).